doc/api/stream.md

```diff
@@ -1273,8 +1273,8 @@ If the `decodeStrings` property is set in the constructor options, then
 indicate the character encoding of the string. This is to support
 implementations that have an optimized handling for certain string
 data encodings. If the `decodeStrings` property is explicitly set to `false`,
-the `encoding` argument can be safely ignored, and `chunk` will always be a
-`Buffer`.
+the `encoding` argument can be safely ignored, and `chunk` will remain the same
+type from the source.
```

Comment:
@fmy you are right. I would phrase that differently thought, because `Writable`  can be used also without `pipe()`. How about:

will remain the same object that is passed to `.write()`.